sunpy/net/vso/vso.py

```diff
@@ -419,6 +419,14 @@ def fetch(self, query_response, path=None, methods=None, site=None,
 
     def make_getdatarequest(self, response, methods=None, info=None):
         """ Make datarequest with methods from response. """
+        
+        #    Pass back the Apache session ID to the VSO if it exists in the response
+
+        for item in response:
+           for k in item.keys():
+              if k == 'Info Required':
+                 info['required'] = item['Info Required']
```

Comment:
A better way to write this would be:
```suggestion
        for item in response:
           info_required = item.get("Info Required", None)
           if info_required is not None:
                 info['required'] = item['Info Required']
```
as it prevents you looping through the keys.

This is assuming you can't just set the required value, i.e. will VSO ignore it if it's there and it's `None`?